src/python/gudhi/simplex_tree.pyx

```diff
@@ -427,11 +427,11 @@ cdef class SimplexTree:
             0.0.
             Sets min_persistence to -1.0 to see all values.
         :type min_persistence: float.
-        :returns: A list of four persistence diagrams in the format described in :func:`persistence()<gudhi.SimplexTree.persistence>`. The first one is Ordinary, the second one is Relative, the third one is Extended+ and the fourth one is Extended-. See https://link.springer.com/article/10.1007/s10208-008-9027-z and/or section 2.2 in https://link.springer.com/article/10.1007/s10208-017-9370-z for a description of these subtypes.
+        :returns: A list of four persistence diagrams in the format described in :func:`persistence`. The first one is Ordinary, the second one is Relative, the third one is Extended+ and the fourth one is Extended-. See https://link.springer.com/article/10.1007/s10208-008-9027-z and/or section 2.2 in https://link.springer.com/article/10.1007/s10208-017-9370-z for a description of these subtypes.
```

Comment:
A bibtex link would be better, no ?

Reply:
Maybe, but that's independent of this patch. The docstring for SimplexTree already has an inline citation for Clément's paper, so it is not the only place. One possible reason not to use bibtex in the python files (I don't know if that's why people did it) is that the text is not just used to generate the sphinx doc, where bibtex runs, it also appears when you do `help(gudhi.SimplexTree)` in python, where there is no bibtex. That may not be a good enough reason not to use bibtex, but I'd say, separate issue/PR to convert them all at once (if people agree that we should change them).